client/src/services/auth/authService.js

```diff
@@ -0,0 +1,18 @@
+import requestService from '@/services/requestService';
+
+const authService = {
+    async signIn(userLoginData) {
+        return await requestService.post('/auth/login', userLoginData);
```

Comment:
after you got success response save token to localStorage, via method saveToken(token) in this service. You don't have to save token to state. Also, you don't need token_type (it is always Bearer) and expiresIn (refresh should be done after you got respective response), so token is enough. Also, add method getToken, which will retrieve token from localStorage.

Reply:
or setToken, as you wish